assets/css/civicrm-admin-utilities-admin.css

```diff
@@ -5272,6 +5272,10 @@ body #crm-container div.crm-summary-row div.crm-label
 	word-break: break-word;
 }
 
+body #crm-container div.crm-summary-row.primary div.crm-label::after {
+        content:" *";
```

Comment:
Could you change this to `content: "*";` please? Keeps the code style intact and AFAICT the extra space isn't necessary since there's already whitespace inside every `<label>` element. If you're concerned that there might not be, an additional `margin-left: 0.1em;` would be a better solution.

Reply:
Other than that, happy to merge with thanks 👍

Reply:
Done, probably didn't do the PR right but I try :)